google/cloud/storage/internal/object_streambuf_test.cc

```diff
@@ -341,6 +341,95 @@ TEST(ObjectWriteStreambufTest, CreatedForFinalizedUpload) {
   EXPECT_EQ(close_result.value().payload, payload);
 }
 
+/// @test Verify that last error status is accessible for small payload.
+TEST(ObjectWriteStreambufTest, erroneousStream) {
+  auto mock = google::cloud::internal::make_unique<
+      testing::MockResumableUploadSession>();
+  EXPECT_CALL(*mock, done).WillRepeatedly(Return(false));
+
+  auto const quantum = UploadChunkRequest::kChunkSizeQuantum;
+  std::string const payload = "small test payload";
+
+  int count = 0;
+  EXPECT_CALL(*mock, UploadFinalChunk(_, _))
+      .WillOnce(Invoke([&](std::string const& p, std::uint64_t s) {
```

Comment:
`s` as the name for an integer may mislead people into thinking it's a string. How about `n`?

Reply:
Changed to n.